### ISSUE TITLE
Send sails compatible socket messages from Automattic/socket.io-emitter

### ISSUE THREAD

**Issue by AnalogJ** [balderdashy/sails#1986]
I'm attempting to build a background processing application that runs as a completely separate process from sails (allowing me to easily scale the workers up and down, and have them run on completely separate machines). 

I've been able to get my models loaded and working using Waterline. I'm working now on getting my background application to send updates to sails, via redis. Is there a specific format that the sails socket messages should be in? 

I've configured redis and Im attempting to send messages using Automattic/socket.io-emitter@ebbc274341c8539016f7a0ded02c05d8a298e744 but I just can't get my messages to look like the messages generated by sails. 

Any help would be appreciated.

**Comment by sailsbot** [comment]
Thanks for posting, @AnalogJ.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
